BrainPortal/app/views/data_providers/_data_providers_table.html.erb

```diff
@@ -33,6 +33,8 @@
     <%= link_to "Create New Data Provider", new_data_provider_path, :class => "button menu_button" %>
   <% end %>
 
+  <%= link_to "Check All", nil, {:class => "button check_all_dp",} %>
```

Comment:
Behavior: if I have already checked manually some data providers in my page using their 'check' link, and I click on 'Check All', they will be rechecked again, even though the information is already on the page.